platform/viewer/src/connectedComponents/ViewerMain.js

```diff
@@ -60,11 +60,12 @@ class ViewerMain extends Component {
     if (this.props.studies) {
       const displaySets = this.getDisplaySets(this.props.studies);
 
-      this.setState({
-        displaySets,
-      });
-
-      this.fillEmptyViewportPanes();
+      this.setState(
+        {
+          displaySets,
+        },
```

Comment:
Do we have different lint settings somehow? I don't get this same change on save.

Reply:
I didn't change my linting settings... no matter what I do, after saving I only get this... :-/
I'm curious how it looks on your end...

Reply:
Actually, if I keep everything in a single line, the linter does not frown on me... :-)
I'll be updating the review soon...